radio/src/translations/untranslated.h

```diff
@@ -129,5 +129,5 @@
 #define LEN_DSM_PROTOCOLS              "\004"
 #define TR_DSM_PROTOCOLS               "LP45""DSM2""DSMX"
 
-#define LEN_MULTI_PROTOCOLS            "\007"
-#define TR_MULTI_PROTOCOLS             "FlySky\0""Hubsan\0""FrSky\0 ""Hisky\0 ""V2x2\0  ""DSM\0   ""Devo\0  ""YD717\0 ""KN\0    ""SymaX\0 ""SLT\0   ""CX10\0  ""CG023\0 ""Bayang\0""ESky\0  ""MT99XX\0""MJXq\0  ""Shenqi\0""FY326\0 ""SFHSS\0 ""J6 PRO\0""FQ777\0 ""Assan\0 ""Hontai\0""OpenLrs""FS 2A\0 ""Q2x2\0  ""Walk.\0 ""Q303\0  ""GW008\0 ""DM002\0 ""Cabell\0""Esy150\0""H8 3D\0 ""Corona\0""CFlie\0 ""Hitec\0 ""WFly\0  ""Bugs\0  ""BugMin\0""Traxas\0""NC1701\0""E01X\0  ""V911S\0 ""GD00X\0 ""V761\0  ""KF606\0 ""Redpin\0""Potens\0""ZSX\0   ""Flyzone"
+#define LEN_MULTI_PROTOCOLS            "\006"
+#define TR_MULTI_PROTOCOLS             "FlySky""Hubsan""FrSky\0""Hisky\0""V2x2\0 ""DSM\0  ""Devo\0 ""YD717\0""KN\0   ""SymaX\0""SLT\0  ""CX10\0 ""CG023\0""Bayang""ESky\0 ""MT99XX""MJXq\0 ""Shenqi""FY326\0""SFHSS\0""J6 PRO""FQ777\0""Assan\0""Hontai""OpnLrs""FS 2A\0""Q2x2\0 ""Walk.\0""Q303\0 ""GW008\0""DM002\0""Cabell""Esy150""H8 3D\0""Corona""CFlie\0""Hitec\0""WFly\0 ""Bugs\0 ""BugMin""Traxas""NC1701""E01X\0 ""V911S\0""GD00X\0""V761\0 ""KF606\0""Redpin""Potens""ZSX\0  ""FlyZon"
```

Comment:
I’m not too fan of having shortened names on radio where it matters more, and different names in companion. Maybe I should make room on radio to display the ‘full’ name ?

Reply:
7 chars are perfect on 128x64 screens, I don't see why we would shorten to 6 chars

Reply:
If we allow more space,we could use the same list as companion : https://github.com/opentx/opentx/blob/2877077fab6320eae25f2d7ce17d3c298690a0f2/companion/src/firmwares/multiprotocols.cpp#L156

Reply:
Using the same table for the options (`multi_protocols` & `multiProtocols`) would be a good idea as well, so they don't have to be kept in sync. It seems already prepared, so why not doing it?

Reply:
Currently on the 128 x 64 screen all items are listed on one line.
eg. Mode  MULT FS_2A  PWM,IBUS

5 Spaces are allotted for module type
6 Spaces are allotted for Protocol
8 Spaces can be used for Sub Protocol with the last character partially hidden by running off screen

In the narrower screen, as on the QX7 or X-Lite, the Subtype is shown on a separate line

I am just trying to work within the boundaries that are currently available?
If more space is allotted then we could definitely have more descriptive names.

Reply:
> Currently on the 128 x 64 screen all items are listed on one line. eg. Mode MULT FS_2A PWM,IBUS

This is on 2.3 (X9 lite):

![IMG_6526](https://user-images.githubusercontent.com/1050031/63305094-d34f4e00-c2e5-11e9-8fde-678821404b56.jpg)

Reply:
My mistake
I meant the 212 x 64 screen everything is on one line. (now edited to correct)

The 128 x 64 is the narrower screen I refer to above of the QX7, X-Lite (and X9 lite)